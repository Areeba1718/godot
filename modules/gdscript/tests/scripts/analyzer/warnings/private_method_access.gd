func test():
    var instance := TestClass.new()
    instance.public()
    instance._private()


class TestClass:

    func public():
        pass
    
    func _private():
        pass
