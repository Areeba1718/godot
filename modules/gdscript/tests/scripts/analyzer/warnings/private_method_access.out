GDTEST_OK
>> WARNING
>> Line: 4
>> PRIVATE_METHOD_ACCESS
>> The method "_private" begins with an underscore and is considered by convention to be private
